WooCommerce/WooCommerceTests/ViewRelated/Dashboard/LocalAnnouncements/LocalAnnouncementViewModelTests.swift

```diff
@@ -0,0 +1,62 @@
+import XCTest
+import Yosemite
+@testable import WooCommerce
+
+final class LocalAnnouncementViewModelTests: XCTestCase {
+    private var analytics: Analytics!
+    private var analyticsProvider: MockAnalyticsProvider!
+    private var stores: MockStoresManager!
```

Comment:
Nit: `stores` doesn't seem to be used, but I suppose you intend to add other tests using it later.

Reply:
I'm not sure how pushing a commit to the base branch ([this base PR](https://github.com/woocommerce/woocommerce-ios/pull/10119)) merged this PR 😕 I plan to add some test cases for the store action today so that we don't leave dead code around.